content/docs/fedora-releases-guide.md

```diff
@@ -0,0 +1,138 @@
+---
+title: How to do Fedora releases with Packit
+weight: 9
+---
+# How to do Fedora releases with Packit
+
+Let's split the release process into single steps:
+1. New upstream release 
+2. Uploading archive to lookaside cache
+3. Updating dist-git content 
+4. Koji builds
+5. Bodhi updates
+
+Doing Fedora releases with Packit means utilising our 3 jobs - `propose_downstream`, `koji_build` and 
+`bodhi_update` - where each of the jobs takes care of a different part of the release process.
+
+## Propose downstream job
+For enabling the propose downstream job, you need to have [Packit Service installed](/docs/packit-service) 
+and have a `propose_downstream` job in the configuration file for the given upstream repository.
+The [propose_downstream job](/docs/configuration/#propose_downstream) should be then configured like this:
+
+```yaml
+jobs:
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - main
+```
+You can adjust the `dist_git_branches` field to include the
+dist-git branches you want to update and also utilise [aliases](/docs/configuration/#aliases) 
+instead of using hardcoded versions.
+
+#### New upstream release
+The process of releasing a new version starts in the upstream repository by creating a 
+new upstream release. Packit gets the information about the newly created release from GitHub,
```

Comment:
```suggestion
new upstream release. Packit gets the information about the newly created release from GitHub (not from a git tag),
```
I know there were questions in the past where folks expected that a git-tag would be fine to do.
So I wonder if this wording makes sense.

Reply:
I included it in that sentence, let me know if it makes sense like this